checkbox-ng/checkbox_ng/resume_menu.py

```diff
@@ -229,7 +229,7 @@ def _create_comment_view(self):
             "body",
         )
 
-    def run(self):
+    def run(self) -> ResumeOutcome:
```

Comment:
Stupid question, but do every version of Python we have to support support this?

Reply:
Not a stupid question at all.
This type of hinting was defined in pep484, and it landed in 3.5 so we're safe here.